### PR TITLE
Fix some typos and bad English in the JSR-352 documentation

### PR DESCRIPTION
**Pull request by yrodiere**

### CODE REVIEW

File: documentation/src/main/asciidoc/manual-index.asciidoc [hunk 1]
```diff
@@ -481,7 +481,7 @@ will attempt to preload everything in memory.
 |-
 |Use HQL / JPQL to index entities of a target entity type. Your query should contain only one entity
 type. Mixing this approach with the criteria restriction is not allowed. Please notice that there's
-no query validation for your input. See <<jsr-352-index-scope>> for more detail and limitations.
```
Comment (Sanne) [inline documentation/src/main/asciidoc/manual-index.asciidoc]: Something is wrong with the links this generates. Missing label?

Comment (yrodiere) [reply documentation/src/main/asciidoc/manual-index.asciidoc]: The anchors were in the wrong place. I added a commit to fix that, too.

**Comment by Sanne** [comment]
great, thanks!

**Comment by Sanne** [comment]
[merged]

**Pull request closed without merging**